packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -515,5 +520,90 @@ describe('masternode', () => {
+      const payload: UtxosToAccountPayload = {}
+      // NOTE(jingyi2811): Only support sending utxos to DFI account.
+      payload['2Mywjs9zEU4NtLknXQJZgozaxMvPn2Bb3qz'] = '5@DFI'
+
+      const utxos = await container.call('listunspent')
+      const inputs = utxos.map((utxo: { txid: string, vout: number }) => {
+        return {
+          txid: utxo.txid,
+          vout: utxo.vout
+        }
+      })
+
+      const promise = client.account.utxosToAccount(payload, inputs)
+
+      await expect(promise).rejects.toThrow(RpcApiError)
+      await expect(promise).rejects.toThrow('The address (2Mywjs9zEU4NtLknXQJZgozaxMvPn2Bb3qz) is not your own address')
+    })
+  })
+
+  describe('accountToAccount', () => {
+    it('should accountToAccount', async () => {
+      const from = await container.getNewAddress()
+      await createToken(from, 'DABC', 5)
+      await createToken(from, 'DDEF', 5)
+
+      const payload: AccountToAccountPayload = {}
+      payload[await container.getNewAddress()] = '5@DFI'
+      payload[await container.getNewAddress()] = '5@DABC'
+      payload[await container.getNewAddress()] = '5@DDEF'
+
+      const data = await client.account.accountToAccount(from, payload)
+
+      expect(typeof data).toStrictEqual('string')
+      expect(data.length).toStrictEqual(64)
+    })
+
+    it('should accountToAccount with utxos', async () => {
+      const from = await container.getNewAddress()
+      await createToken(from, 'DGHI', 5)
+      await createToken(from, 'DJKL', 5)
+
+      const { txid } = await container.fundAddress(from, 10)
+
+      const payload: AccountToAccountPayload = {}
+      payload[await container.getNewAddress()] = '5@DFI'
+      payload[await container.getNewAddress()] = '5@DGHI'
+      payload[await container.getNewAddress()] = '5@DJKL'
+
+      const utxos = await container.call('listunspent')
+      const inputs = utxos.filter((utxo: { txid: string, vout: number }) => utxo.txid === txid).map((utxo: any) => {
+        return {
+          txid: utxo.txid,
+          vout: utxo.vout
+        }
+      })
+
+      const data = await client.account.accountToAccount(from, payload, inputs)
+
+      expect(typeof data).toStrictEqual('string')
+      expect(data.length).toStrictEqual(64)
+    })
+
+    it('should not accountToAccount with utxos for DFI coin if does not own the recipient address', async () => {
+      const from = await container.getNewAddress()
+
+      const { txid } = await container.fundAddress(from, 10)
+
+      const payload: AccountToAccountPayload = {}
+      payload['2Mywjs9zEU4NtLknXQJZgozaxMvPn2Bb3qz'] = '5@DFI'
+
+      const utxos = await container.call('listunspent')
+      const inputs = utxos.filter((utxo: { txid: string, vout: number }) => utxo.txid === txid).map((utxo: any) => {
+        return {
+          txid: utxo.txid,
+          vout: utxo.vout
+        }
+      })
```

Comment:
do you need to provide inputs utxo for failure test? if you don't need it. should not add it.

Reply:
The point is your test should be self-sufficient to test that single error behaviour without noise.